content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,266 @@
+    </Box>
+    <Box mb={3} display="flex" alignItems="center" flexDirection="column" mx="auto" sx={{gap: 2}}>
+        <img
+            width="456"
+            src="https://user-images.githubusercontent.com/2313998/152432688-e74f67b2-a777-4c5f-be45-37a43615e6ab.png"
+        />
+        <Text as="p" mt="0" align="center" fontSize={1}>Inputs divided into multiple sections</Text>
+    </Box>
+</Box>
+
+### State
+
+Do not disable or hide a form's save button even if the form is invalid or has not been changed. Disabled buttons cannot be focused using the Tab key, and disabled button styles are typically low-contrast and difficult to read.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://user-images.githubusercontent.com/2313998/151623709-2a4257d6-e8d0-4156-b928-c7097981791c.png" />
+    <Caption>Always keep the save button enabled</Caption>
+  </Do>
+  <Dont>
+    <img src="https://user-images.githubusercontent.com/2313998/151623715-3b298004-b624-43dc-875c-2590c08070e7.png" />
+    <Caption>Don't disable the save button if the form is invalid or unchanged</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Placement
+
+Place save buttons somewhere intuitive and easily accessible. If all of the fields save automatically, do not include a save button. This makes it unclear when data is being saved. See the guidelines on [explicit saving](#explicit-saving) and [automatic saving](#automatic-saving) for more information.
+
+- There should only be one save button on a page. 
+- Place save buttons somewhere intuitive and easily accessible. If all of the fields save automatically, do not include a save button. This makes it unclear when data is being saved. See the guidelines on [explicit saving](#explicit-saving) and [automatic saving](#automatic-saving) for more information.
+- Allow only one edit mode activated at a time if you're designing a page with content that can be edited separately (such as an issue title). Pages that contain multiple save buttons can cause confusion about which save button saves which group. Multiple save buttons with the same label can cause confusion for people who use assistive technologies like screen readers and voice recognition software.
+
+<DoDontContainer>
+  <Do>
+    <img alt="A form with one save button at the bottom" src="https://user-images.githubusercontent.com/2313998/151887934-cfcc5130-53a8-4f40-ba36-97148ead72c0.png" />
+    <Caption>Have one save button per form</Caption>
+  </Do>
+  <Dont>
+    <img alt="A form with two sections. Each section has it's own save button." src="https://user-images.githubusercontent.com/2313998/151887938-1fa6b5d7-c5c1-4010-ab99-e5c37bff8c0a.png" />
+    <Caption>Don't have one save button per section of a form</Caption>
+  </Dont>
+</DoDontContainer>
+
+#### Bottom-left
+
+If there is a button to submit and a button to cancel, the cancel button should go to the **right** of the submit button.
```

Comment:
This statement confused me, because I don't see a cancel button on the example. 🤔

Reply:
Great point. I'll update the example.